.github/workflows/pre-commit-validation.yml

```
@@ -17,199 +17,130 @@ jobs:
+            echo "commits<<EOF"
+            echo "$COMMITS"
+            echo "EOF"
+            echo "files<<EOF"
+            echo "$FILES"
+            echo "EOF"
+          } >> "$GITHUB_OUTPUT"
+
       - name: Validate branch naming convention
         if: github.event_name == 'pull_request'
+        shell: bash
         run: |
```

actionlint (1.7.11)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)